### DISCUSSION TITLE
Bonus T3

### DISCUSSION THREAD

**Discussion by JavierLarre** [IIC2133-PUC/2024-1#174] [Tarea 3] [answered]
Hola, hay planeado un bonus en la T3 igual que en la T2 o T1? muchas gracias
PD: y va a haber restricción de complejidad para alguna parte?

**Comment by sachondo7** [comment, marked as answer]
Nop, esta tarea no tiene bonus. 
Lo de complejidad debe estar de acuerdo a lo que se enseña en el curso, no pueden chantarle algo tipo n! n^3, 2^2, etc. Pero no, no se pide alguna en especifico, preocupate que corra antes de los 10s

**Reply by MaxAl100** [reply]
A qué se refieren con 2^2? 😅

**Reply by sachondo7** [reply]
nada, no pesques.
